dynamo/tools/recipes.py

```diff
@@ -17,8 +17,8 @@ def recipe_kin_data(
     X_total_layers=False,
     splicing_total_layers=False,
     n_top_genes=1000,
-    keep_filtered_cells=False,
-    keep_filtered_genes=keep_filtered_genes,
+    keep_filtered_cells=None,
```

Comment:
I would keep the default of filtered_cells, genes as before because this is a receipe which is supposed to choose some favorable setups for the analysis. 

You could add a data_store_mode argument to this function, default to None, but  if user set that, follow the setup in dynamo configuration; otherwise, use the previous defaults in keep_filtered_genes, keep_raw_layers, del_2nd_moments

You could add a log inform to explain this logic

Reply:
Use case: users can change the behaviors of functions by changing the mode in config once. Or we can add data_store_mode to every function and let users do extra work. We can have a default mode for data_store_mode to store these favored cases.

Reply:
add those to the default recipe mode in the configuration will be better. Your current changes will break a lot of my analysis done previosuly.

Reply:
Ok. I have added those to configuration files.

Reply:
only need to provide the recipe related functions' good defaults